Client/Frontend/Fakespot/Components/CardContainer.swift

```diff
@@ -0,0 +1,78 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import Shared
```

Comment:
What is Shared used for? Just so we make sure we can move this component easily later

Reply:
Damn, I thought I had removed it already.